tgg/seqr_scripts/sex_check.py

```diff
@@ -0,0 +1,208 @@
+import argparse
+import logging
+import matplotlib.pyplot as plt
+import pandas as pd
+from os.path import dirname
+
+import hail as hl
+import hail.expr.aggregators as agg
+
+from gnomad.utils.reference_genome import get_reference_genome
+
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s", 
+    datefmt="%m/%d/%Y %I:%M:%S %p", 
+)
+logger = logging.getLogger("sex_check")
+logger.setLevel(logging.INFO) 
+
+
+def get_chr_cov(mt: hl.MatrixTable, build: str, call_rate_threshold: float=0.25, chr_name: str, af_field: str="AF", af_threshold: float=0.01) -> hl.Table:
+    """
+    Calculate mean chromosome coverage. 
+
+    :param mt: MatrixTable containing samples with chrY variants
+    :param build: Reference used, either GRCh37 or GRCh38
+    :param call_rate_threshold: Minimum required call rate
+    :param chr_name: chosen chromosome, either 1, 2, 3, ..., X, or Y
+    :param af_field: allele frequency field of matrix table 
+    :param af_threshold: allele frequency threshold 
+    :return: Table with coverage annotations
```

Comment:
```suggestion
    :param chr_name: Chosen chromosome. Must be either autosome (number only) or sex chromosome (X, Y)
    :param af_field: Name of field containing allele frequency information. Default is "AF"
    :param call_rate_threshold: Minimum call rate threshold. Default is 0.25
    :param af_threshold: Minimum allele frequency threshold. Default is 0.01
    :return: Table with coverage annotations
```
some small suggestions and reordering to match the order the arguments are defined